### ISSUE TITLE
Release 0.4.2 version

### ISSUE THREAD

**Issue by nchashch** [amethyst/amethyst#202]
There was some work on improving docs and some non breaking bug fixes on `develop` branch. I think it makes sense to release `0.4.2` version after #201 is merged and appveyor build failure is fixed.

Releasing early would allow us to have a more detailed and helpful changelog/development history. Also better docs/bug fixes would become available on crates.io.

Current todo for `0.4.2`:
- [x] fix appveyor build failure
- [x] merge #201

Coment here if you want to make some more small+non breaking changes before release.

#152, #181, #186, #193, #195 when they are ready would go into `0.5.0` release.

**Comment by ebkalderon** [comment]
Version 0.4.2 has been released! We can close this now. I will start using milestones to keep track of release 0.5.0 so we don't need to create a new issue every time.
